lib/display.rb

```diff
@@ -83,4 +83,26 @@ def show_command_instructions
   def exit_or_proceed(input)
     input == 'exit' ? exit(true) : input
   end
+
+  def enter_another_command
+    puts 'Feel free to enter another set of commands'
```

Comment:
why `_another_` why not just `enter_commad`

that said, how is this enter command? this is notify user to enter command. This makes it feel like it should just be a `notify(str)` command and the string actually belongs to the application whether it is
* 'Feel free to enter another set of commands'
* or 'you have found the jackpot'
* or what ever